mist-common/src/main/java/edu/snu/mist/common/operators/AggregateWindowOperator.java

```diff
@@ -82,4 +83,9 @@ public void processLeftData(final MistDataEvent input) {
   public void processLeftWatermark(final MistWatermarkEvent input) {
     outputEmitter.emitWatermark(input);
   }
+
+  @Override
+  public void processLeftCheckpoint(final MistCheckpointEvent input) {
+    outputEmitter.emitCheckpoint(input);
```

Comment:
Don't we checkpoint the state of this operator?

Reply:
No. This operator was not considered as stateful.

Reply:
I don't think it is necessary because the purpose of this operator is to aggregate the data from one window, and it should not keep a state(that is the result of previous windows).